value.go

```diff
@@ -37,6 +38,10 @@ func ValueOf(i interface{}) Value {
 
 // valueOf recursively returns a new value.
 func valueOf(v reflect.Value) js.Value {
+	if t, ok := v.Interface().(time.Time); ok {
```

Comment:
Can this be pushed down into the `valueOfStruct` logic since the kind is a struct?

Reply:
fine for me ... will do later.